composer.json

```diff
@@ -15,12 +15,12 @@
     "require": {
         "php": "^5.5 || ^7.0",
         "ext-mbstring": "*",
```

Comment:
Nevermind, I was reading that as "mcrypt", when it's in fact "mbstring". Nothing to do here, move along...